src/components/admin/store/PickupOrdersPrepareDisplay/index.tsx

```diff
@@ -66,9 +68,9 @@ const PickupOrdersPrepareDisplay = ({ orders }: PickupOrdersDisplayPrepareProps)
                   <ul className={styles.itemList}>
                     {itemQuantities.map(item => (
                       <li key={item.uuid}>
-                        <Typography variant="h5/regular">{`${item.quantity} x ${itemToString(
-                          item
-                        )}`}</Typography>
+                        <Typography variant="h5/regular">{`${item.fulfilled ? '✅' : '❌'} ${
```

Comment:
It might be nice to add a title attribute to these explaining what the emoji means

Reply:
At least for the user facing orders

Reply:
i removed for user facing 😁 
I assume since we're implementing as checkboxes in #244 we won't need these later so I'll just keep for now